src/components/FilterTabs/FilterTab.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChakraComponent, useStyleConfig, Button } from "@chakra-ui/react"
+import { useContext } from "react"
+import { FilterTabListContext } from "./context"
+
+export interface FilterTabComponent
+  extends ChakraComponent<"button", { tabId: string }> {}
+
+export const FilterTab: FilterTabComponent = ({
+  tabId,
+  children,
+  ...props
+}) => {
+  const { selectedTabId, variant, onTabClick } =
+    useContext(FilterTabListContext)
```

Comment:
Can we create a wrapper for it in `/context/FilterTabListContext`? I mean:
```
const useFilterTablListContext = () => {
 return useContext(FilterTabListContext)
}
```

Reply:
Fixed in c7ec175db8710f0213c8d3e9f35d8d2527371d93